pygotham/about/models.py

```diff
@@ -0,0 +1,29 @@
+"""About models."""
+
+from slugify import slugify
+from sqlalchemy_utils.decorators import generates
+
+from pygotham.core import db
+
+__all__ = ('AboutPage',)
+
+
+class AboutPage(db.Model):
```

Comment:
Because I was so bad about it in the past, one thing we need to do is link `AboutPage` and `Event`. I'd suggest adding `event_id` to `AboutPage` and then adding something to the admin to allow pages to be copied from one event to another.